lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -239,6 +239,22 @@ def fetch_channel_istate(channel)
           channel.connection.socket.istate
         end
       end
+
+      def full_subscription_prefix
+        "#{config.redis_prefix}-#{SUBSCRIPTION_PREFIX}"
+      end
+
+      def full_fingerprints_prefix
+        "#{config.redis_prefix}-#{FINGERPRINTS_PREFIX}"
+      end
+
+      def full_subscriptions_prefix
+        "#{config.redis_prefix}-#{SUBSCRIPTIONS_PREFIX}"
+      end
+
+      def full_channel_prefix
+        "#{config.redis_prefix}-#{CHANNEL_PREFIX}"
+      end
```

Comment:
Let's create a single helper method, say, `#redis_key(val)`, which just adds a prefix, instead of having N almost identical ones.

Reply:
Oh yeah, it is an obvious and logical solution
Why I didn`t do it before, I do not know

Thanks!
fixed